### ISSUE TITLE
Desert and Nether storms invisible with mod "Fancy block particles"

### ISSUE THREAD

**Issue by HarumiAme** [OreCruncher/DynamicSurroundings#193]
*Mod Version:*
DynamicSurroundings-1.10.2-3.4.7.2

*Forge Version:*
1.10.2 - 12.18.3.2511 

*Description:*
When using the mod "Fancy block particles" (One of the suggested companions) (https://minecraft.curseforge.com/projects/fancy-block-particles) sand storms are invisible. The only visual effect that appears is the fog color.

**Comment by OreCruncher** [comment]
Must have been a "recent" change to break this.

**Comment by OreCruncher** [comment]
Yep - recent change.  Turn off the fancy rain and snow in FBP (the settings are in the Particle.properties file in the FBP folder) and it should work.  FBP is hooking the weather renderer when this is enabled and that blocks DS from doing it's thing.  Unfortunately this is something that I cannot fix.

**Comment by TominoCZ** [comment]
Oh hey there.. I was browsing the internet and happened to come across this post.. 
I mean yeah.. it would be great if there was a list of weather renderers to which you could just add your renderer.

**Comment by OreCruncher** [comment]
I was thinking the same thing. :)

**Comment by darthvader45** [comment]
That does sound like something Forge could do.  Maybe pop a message over to their github?
